lib/vsc/install/shared_setup.py

```diff
@@ -20,23 +20,46 @@
 @author: Andy Georges (Ghent University)
 """
 import glob
+import inspect
 import os
 import shutil
 import sys
 import re
+
 from distutils import log  # also for setuptools
 from distutils.dir_util import remove_tree
 
+from setuptools.command.test import test as TestCommand
+
+
 # 0 : WARN (default), 1 : INFO, 2 : DEBUG
 log.set_verbosity(2)
 
 has_setuptools = None
 
+# available authors
+ag = ('Andy Georges', 'andy.georges@ugent.be')
+jt = ('Jens Timmermans', 'jens.timmermans@ugent.be')
+kh = ('Kenneth Hoste', 'kenneth.hoste@ugent.be')
+lm = ('Luis Fernando Munoz Meji?as', 'luis.munoz@ugent.be')
+sdw = ('Stijn De Weirdt', 'stijn.deweirdt@ugent.be')
+wdp = ('Wouter Depypere', 'wouter.depypere@ugent.be')
+kw = ('Kenneth Waegeman', 'Kenneth.Waegeman@UGent.be')
+eh = ('Ewan Higgs', 'Ewan.Higgs@UGent.be')
+wp = ('Ward Poelmans', 'Ward.Poelmans@UGent.be')
+
+# FIXME: do we need this here? it won;t hurt, but still ...
+REGEXP_REMOVE_SUFFIX = re.compile(r'(\.(?:py|sh|pl))$')
```

Comment:
If we don't mind splitting any ends, we can use `'some-string'.rsplit('.', 1)[0]`